source/clear-linux/guides/maintenance/kernel-development.rst

```diff
@@ -0,0 +1,365 @@
+.. _kernel-development:
+
+Kernel development on Clear Linux 
+#################################
+
+The kernel is the heart of an operating system with many things relying on it
+to provide access to hardware and software functionality. 
+The Linux kernel is very customizable and extensible. 
+ 
+The |CL| kernels aim to be performant and practical. In some cases, it may be 
+necessary modify the kernel to suit your specific needs or test new kernel 
+code as a developer.
+
+In cases where features beyond those enabled by default in |CL-ATTR| are
+needed it may be necessary to:
+
+.. contents:: :local:
+   :depth: 2
+   :backlinks: top
+
+
+Request the change be included with the |CL| kernel
+===================================================
+
+If the kernel modification you need is already open source and likely to be 
+useful to others, consider submitting a request to add or enable in the
+|CL| kernel.
+
+If your change request is accepted, you do not need to maintain your own 
+modified kernel.
+
+Make enhancement requests to the |CL| distribution `on GitHub`_ .
+
+
+Customize the Linux kernel source
+=================================
+
+In some cases, it may be necessary modify the kernel to suit your specific 
+needs or test new kernel code as a developer.
+
+You can build and install a custom kernel, however you must:
+
+* Disable Secure Boot
+* Maintain any updates to the kernel going forward
+
+To customize the kernel you will need a |CL| development environment, 
+make changes to the kernel, build the kernel, and install it.
+
+.. note::
+
+      This document shows how to obtain and compile a Linux kernel source 
```

Comment:
This line should probably be part of the introduction to this document and doesn't need to be a Note.